aws-lambda/src/main.rs

```diff
@@ -14,7 +14,8 @@ mod runtime;
 #[tokio::main]
 async fn main() -> Result<(), Error> {
     tracing_subscriber::fmt()
-        .with_max_level(tracing::Level::WARN)
+        // log everything here since logs can be filtered by level in CloudWatch.
+        .with_max_level(tracing::Level::TRACE)
```

Comment:
Log Levels are set by env variables. Why do we need this?

Reply:
Lambda logs are not set by env variables. It logs to AWS CloudFront. Every log should be ingested into CloudFront, where it can later be filtered, that is why I've configured the tracing subscriber to log everything.

Reply:
In either case, I think this should be configured by the user and we shouldn't hard code it. May be via our terraform scripts.

Reply:
I can add an env variable. This or next PR?

Reply:
did it in this PR.